src/View/Resource/index.html.twig

```diff
@@ -1,10 +1,13 @@
 {% extends 'layout.html.twig' %}
-{% block title %}{{parent()}} -Ressource{% endblock %}
+{% block title %}
+	{{parent()}}
+	|Ressource
+{% endblock %}
 {% block stylesheet %}
 	<link rel="stylesheet" href="assets/css/faq.css">
 {% endblock %}
 {% block content %}
-<h1>FAQ</h1>
+	<h1>FAQ</h1>
 	{% for item in faq %}
```

Comment:
pas bien nommer
devrait etre for "trucAuSingulier" in "trucAuPluriel"
c'est pas ouf faqs (mais déjà faq c'est pas terrible comme nom vu que c'est une abbréviation :D )
bref pour limiter les pb je ferais quand meme un for faq in faqs en renommant $faq en $faqs dans le controller